src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -29,6 +29,7 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
+import java.util.Enumeration;
```

Comment:
Better put `import java.util.Enumeration;` one row above

Reply:
I don't see why would we need to move `Enumeration`, right now it's in alphabetical order (Collections, Enumeration, List).

Reply:
Yep, sorry